Source/Scene/PointAttenuationOptions.js

```diff
@@ -0,0 +1,71 @@
+define([
+    '../Core/defaultValue'
+], function(
+    defaultValue
+) {
+    'use strict';
+
+    /**
+     * Options for performing point attenuation based on geometric error when rendering
+     * pointclouds using 3D Tiles.
+     *
+     * @param {Object} [options] Object with the following properties:
+     * {Boolean} [options.geometricErrorAttenuation=false] Perform point attenuation based on geometric error.
+     * {Number} [options.geometricErrorScale=1.0] Scale to be applied to each tile's geometric error.
+     * {Number} [options.maximumAttenuation] Maximum attenuation in pixels. Defaults to the Cesium3DTileset's maximumScreenSpaceError.
+     * {Number} [options.baseResolution] Average base resolution for the dataset in meters. Substitute for Geometric Error when not available.
+     * {Boolean} [options.eyeDomeLighting=false] When true, use eye dome lighting when drawing with point attenuation.
+     * {Number} [options.eyeDomeLightingStrength=1.0] Increasing this value increases contrast on slopes and edges.
```

Comment:
Here and below, what is the typical range?

Reply:
`eyeDomeLightingStrength` looks best to me between 0.5 and 4.0, otherwise it starts getting too light or too dark..
`eyeDomeLightingRadius` is in pixels, so between 1.0 and 2.0 seems to be a good range.

Reply:
OK, please document.